setup.py

```diff
@@ -37,7 +37,7 @@ def find_version(*file_paths):
     'requests >= 2.6.1, != 2.11.0, < 2.12',
     'texttable >= 0.8.1, < 0.9',
     'websocket-client >= 0.32.0, < 1.0',
-    'docker >= 2.3.0, < 3.0',
+    'docker >= 2.4.2, < 3.0',
```

Comment:
@shin- Taking into account that this may break a lot of builds around the world, perhaps it would be good to be a little bit more restrictive here. I mean, `docker >= 2.4.2, < 2.5.0` or something around these lines. What do you think about it?